### ISSUE TITLE
TypeError: decode() takes no keyword arguments

### ISSUE THREAD

**Issue by kevinrenskers** [invisibleroads/imapIO#2]
Thanks for fixing the previous issue. Importing imapIO works, but running the example code does not:

```
>>> emailGenerator = server.walk(excludes=['public', 'trash'], searchCriterion=emailCriterion)
>>> for emailIndex, email in enumerate(emailGenerator):
...     print email
... 
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/renskers/Workspace/Planbord/lib/python2.6/site-packages/imapIO/__init__.py", line 123, in walk
    yield Email(self, messageUID, tags, data[0][1])
  File "/Users/renskers/Workspace/Planbord/lib/python2.6/site-packages/imapIO/__init__.py", line 221, in __init__
    self.subject = self._decode(valueByKey.get('Subject', ''))
  File "/Users/renskers/Workspace/Planbord/lib/python2.6/site-packages/imapIO/__init__.py", line 241, in _decode
    string = ''.join(part.decode(encoding or 'utf-8', errors='ignore') for part, encoding in packs)
  File "/Users/renskers/Workspace/Planbord/lib/python2.6/site-packages/imapIO/__init__.py", line 241, in <genexpr>
    string = ''.join(part.decode(encoding or 'utf-8', errors='ignore') for part, encoding in packs)
TypeError: decode() takes no keyword arguments
```

**Comment by kevinrenskers** [comment]
http://docs.python.org/library/stdtypes.html#str.decode
Changed in version 2.7: Support for keyword arguments added.

**Comment by kevinrenskers** [comment]
Same error on line 425... With those 2 lines fixes, imapIO works on Python 2.6.
